repose-aggregator/components/filters/slf4j-http-logging-filter/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     testCompile "org.apache.logging.log4j:log4j-core"
     testCompile group: 'org.apache.logging.log4j', name: 'log4j-core', classifier: 'tests'
     testCompile "org.apache.logging.log4j:log4j-slf4j-impl"
-    testCompile "com.mockrunner:mockrunner-servlet"
+    testCompile "org.springframework:spring-test"
```

Comment:
Nope. It is either currently being brought in by something else transitively or these tests are not causing anything to actually be logged.